### PR TITLE
Implement custom fields for GetTasks

### PR DESCRIPTION
**Pull request by heldeen**
* add custom fields and their various operators for use w/ the filtering on Get Tasks and Get Filtered Team tasks endpoints
* source of operators is here https://clickup.com/api/developer-portal/filtertasks/

### CODE REVIEW

**Comment by heldeen** [comment]
@w-haibara, I see you being tagged on other PRs. Can you look at this one?

**Comment by heldeen** [comment]
@a-thug @tkowalski29 Please review this simple change that adds custom field searching.

File: clickup/tasks.go [hunk 1]
```diff
@@ -178,22 +179,75 @@ type TaskStatusHistory struct {
...
+
+const (
+	Equals               CustomFieldInGetTasksRequestOperator = iota //=
+	LessThan                                                         // <
+	LessThanOrEqualTo                                                // <=
+	GreaterThan                                                      // > (greater than)
+	GreaterThanOrEqualTo                                             // > = (greater than or equal to)
+	NotEqualTo                                                       // != (not equal to)
+	IsNull                                                           // IS NULL (is not set)
+	ISNotNull                                                        // IS NOT NULL (is set)
```
Comment (a-thug) [inline clickup/tasks.go:221]:
```suggestion
	IsNotNull                                                        // IS NOT NULL (is set)
```

File: clickup/tasks_test.go [hunk 2]
```diff
@@ -122,3 +122,68 @@ func TestTasksService_GetTask(t *testing.T) {
...
+		Reverse:  false,
+		CustomFields: CustomFieldsInGetTasksRequest{
+			{
+				FieldId:  "de761538-8ae0-42e8-91d9-f1a0cdfbd8b5",
+				Operator: GreaterThan,
+				Value:    []string{"2"},
+			},
+			{
+				FieldId:  "4223cfb4-b14b-4bd4-aa35-81ae29c62f4d",
+				Operator: ISNotNull,
```
Comment (a-thug) [inline clickup/tasks_test.go:164]:
```suggestion
				Operator: IsNotNull,
```

**Review by a-thug — COMMENTED** [review]
@heldeen 
I've suggested it, but it looks fine to me.:+1:

@w-haibara 
Could you merge if the suggested part is picked up?

**Review by a-thug — APPROVED** [review]

**Comment by w-haibara** [comment]
@heldeen san, thank you for your contribution! Sorry for the late response.
I'll merge it.

**Pull request merged**